spec/setup_acceptance_node.pp

```diff
@@ -4,6 +4,14 @@
   package { 'glibc-langpack-en':
     ensure => installed,
   }
+
+  # Workaround for https://github.com/theforeman/puppet-candlepin/issues/188
+  package { 'pki-core':
```

Comment:
I think you need to ensure this is done before `candlepin-selinux` is installed or it may fail.